starter-ui/src/test/resources/HttpRequest.http

```diff
@@ -0,0 +1,4 @@
+###
```

Comment:
What is this file for?

Reply:
Ah that is the IntelliJ HttpRequest plugin way of separating commands :-)